ecoscope/io/earthranger.py

```diff
@@ -922,3 +798,94 @@ def patch_event(
         else:
             results = [self._patch(f"activity/event/{event_id}", payload=events)]
         return pd.DataFrame(results)
+
+    def post_subject(
+        self,
+        subject_name: str,
+        subject_type: str,
+        subject_subtype: str,
+        is_active: bool = True,
+        tracks_available: bool = True,
```

Comment:
tracks_available might be an internal parameter. I would suggest we don't let the user try and set this.

Reply:
You're right. I removed the tracks_available parameter.